metricbeat/module/elasticsearch/ccr/_meta/docs.asciidoc

```diff
@@ -1,3 +1,8 @@
 This is the `ccr` metricset of the Elasticsearch module. It interrogates the
-Cross Cluster Replication Stats API endpoint to fetch information about shards
-in the Elasticsearch cluster that are participating in cross-cluster replication.
+Cross-Cluster Replication Stats API endpoint to fetch metrics about cross-cluster
```

Comment:
This is a sentence fragment. Maybe say something like:

"This metricset uses the Cross-Cluster Replication Stats API endpoint to fetch...."